bin/cloudworker.js

```diff
@@ -64,12 +101,22 @@ function run (file, wasmBindings) {
       console.log('Changes to the worker script detected - reloading...')
 
       server.close(() => {
-        if (stopping) return
-
+        if (stopping) {
+          if (httpsServer) {
+            httpsServer.close(() => { })
+          }
+        }
```

Comment:
It looks the `return` was dropped by this change. It's required so that a new server is created if stopping.

Reply:
Done